services/src/main/scala/cromwell/services/metadata/impl/webhook/WebHookMetadataServiceActor.scala

```diff
@@ -0,0 +1,99 @@
+package cromwell.services.metadata.impl.webhook
+
+import akka.actor.{Actor, ActorLogging, ActorRef, ActorSystem, Props}
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model._
+import com.typesafe.config.Config
+import cromwell.core.Dispatcher._
+import cromwell.services.metadata.MetadataService.{MetadataWriteFailure, MetadataWriteSuccess, PutMetadataAction, PutMetadataActionAndRespond}
+import cromwell.services.metadata._
+import cromwell.services.metadata.impl.MetadataServiceActor
+import net.ceedubs.ficus.Ficus._
+import spray.json._
+
+import scala.concurrent.{ExecutionContextExecutor, Future}
+import scala.util.{Failure, Success}
+
+
+class WebHookHybridMetadataServiceActor(serviceConfig: Config, globalConfig: Config, serviceRegistryActor: ActorRef) extends Actor with ActorLogging {
```

Comment:
I believe this part might already be covered by the `TeeingMetadataActor`

Reply:
@cjllanwarne I cannot seem to find any reference to the `TeeingMetadataActor`. Is this an `Akka` thing?